.gitignore

```diff
@@ -3,5 +3,5 @@ obj
 bin
 node_modules
 wwwroot/css
-/appsettings.Development.json
+/NetcoreBootstrap/appsettings.Development.json
```

Comment:
why do you add `/NetcoreBootstrap/` only here?

Reply:
It's only referencing the main appsetings.Development.json. Before, it was referencing the one on the root folder. Since the project structure changed, the file is now on the NetcoreBootstrap folder

Reply:
Oks :)